setup.py

```diff
@@ -75,6 +76,9 @@ def get_static_files(path):
         ]
     },
     install_requires=install_requires,
+    extras_require={
+        'toml': ['toml'],
```

Comment:
What are your thoughts on limiting this to the current milestone version? i.e. `'toml<1.0'` (since the current is 0.9.4) I just don't want version 1.0.0 to break this. I'd rather tread with caution, even with an optional configurable feature.

Reply:
Python toml package version [related to toml spec version](https://github.com/uiri/toml/blob/master/RELEASE.rst). [TOML spec](As of version 0.5.0, TOML should be considered extremely stable. The goal is for version 1.0.0 to be backwards compatible (as much as humanly possible) with version 0.5.0. All implementations are strongly encouraged to become 0.5.0 compatible so that the transition to 1.0.0 will be simple when that happens.) v1.0 will be backward compatible. Some quotes:

> 1.0.0 will be the first version to support TOML 1.0.0. TOML still has yet to hit TOML 1.0.0 so its release is postponed indefinitely.

> As of version 0.5.0, TOML should be considered extremely stable. The goal is for version 1.0.0 to be backwards compatible (as much as humanly possible) with version 0.5.0. All implementations are strongly encouraged to become 0.5.0 compatible so that the transition to 1.0.0 will be simple when that happens.

Reply:
In that case, why don't we put `'toml<2.0'`?

Reply:
Ok. This don't break anything. This fix added. Thank you for review :)